diff --git a/dist/runtime/components/CldImage.vue b/dist/runtime/components/CldImage.vue
index 5f3ebaa..8a7573f 100644
--- a/dist/runtime/components/CldImage.vue
+++ b/dist/runtime/components/CldImage.vue
@@ -113,7 +113,7 @@ const props = defineProps<CldImageProps>();
 // eslint-disable-next-line vue/no-setup-props-destructure
 const { config, loaderOptions, ...options } = props;

-const { url } = useCldImageUrl({ options, config });
+const { url } = useCldImageUrl({ options: options as any, config });

 const transformUrl = () => {
@@ -151,7 +151,7 @@ const transformUrl = () => {
     options.widthResize = loaderOptions?.width;
   }

-  const { url } = useCldImageUrl({ options, config });
+  const { url } = useCldImageUrl({ options: options as any, config });

   return url;
 }
diff --git a/dist/runtime/components/CldOgImage.vue b/dist/runtime/components/CldOgImage.vue
index 44bd93e..cb0b73b 100644
--- a/dist/runtime/components/CldOgImage.vue
+++ b/dist/runtime/components/CldOgImage.vue
@@ -110,14 +110,14 @@ const { url: ogImageUrl } = useCldImageUrl({
   options: {
     ...options,
-    format: props.format || "jpg",
+    format: (props.format || "jpg") as any,
+  } as any,
 });

 const { url: twitterImageUrl } = useCldImageUrl({
   options: {
     ...options,
-    format: props.format || "webp",
+    format: (props.format || "webp") as any,
+  } as any,
 });
diff --git a/dist/runtime/components/CldVideoPlayer.vue b/dist/runtime/components/CldVideoPlayer.vue
index e039b62..049bc3c 100644
--- a/dist/runtime/components/CldVideoPlayer.vue
+++ b/dist/runtime/components/CldVideoPlayer.vue
@@ -3,7 +3,7 @@ import { ref } from "vue";
 import { useHead } from "@unhead/vue";
 import { useRuntimeConfig } from "#imports";
 import { parseUrl } from "@cloudinary-util/util";
-import { ConfigOptions } from "@cloudinary-util/url-loader";
+import type { ConfigOptions } from "@cloudinary-util/url-loader";

 export interface CloudinaryVideoPlayer {
